apps/aecore/lib/aecore/oracle/oracle_state_tree.ex

```diff
@@ -77,23 +77,22 @@ defmodule Aecore.Oracle.OracleStateTree do
   end
 
   defp filter_expired({expires, data}, expires, cache_key_encoded, trees) do
```

Comment:
seems like "expires" is unused here?

Reply:
@d-velev 
We use it at level function pattern matching